src/components/search-form/SearchActions.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import { injectIntl, IntlShape } from 'react-intl';
+
+import ClearBadge16 from '../../icon/fill/ClearBadge16';
+import Search16 from '../../icon/fill/Search16';
+import makeLoadable from '../loading-indicator/makeLoadable';
+
+import messages from './messages';
+
+export interface SearchActionsProps {
+    /** Whether to render an interactive search button */
+    hasSubmitAction: boolean;
+    /** Intl object */
+    intl: IntlShape;
+    /** Called when clear button is clicked */
+    onClear: (event: React.SyntheticEvent<HTMLButtonElement>) => void;
+}
+
+const SearchActions = ({ hasSubmitAction, intl, onClear }: SearchActionsProps) => {
```

Comment:
fyi this component is just copied from `SearchForm`. leaving class names and whatnot as the same so it doesn't break existing functionality for apps